### ISSUE TITLE
What is <Yield> Keyword in Python?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#15]
What is `yield` Keyword in Python?

- The `yield` keyword in Python can turn any function into a generator. Yields work like a standard return keyword.
But it’ll always return a generator object. Also, a function can have multiple calls to the `yield` keyword.

### Example
```python
def testgen(index):
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sat’]
yield weekdays[index]
yield weekdays[index+1]

day = testgen(0)
print next(day), next(day)
```
    Output: sun mon

**Comment by AshishNamdev** [comment]
Included in [Q13](../blob/master/questions/Q1-50.md#Q13)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
